docs/resources/sql_endpoint.md

```diff
@@ -36,14 +36,15 @@ The following arguments are supported:
 * `tags` - Databricks tags all endpoint resources with these tags.
 * `spot_instance_policy` - The spot policy to use for allocating instances to clusters: `COST_OPTIMIZED` or `RELIABILITY_OPTIMIZED`. This field is optional. Default is `COST_OPTIMIZED`.
 * `enable_photon` - Whether to enable [Photon](https://databricks.com/product/delta-engine). This field is optional and is enabled by default.
-* `enable_serverless_compute` - Whether this SQL warehouse is a serverless endpoint. If this value is true explicitly or through the default, you **must** also set `warehouse_type` field to `pro`.
+* `enable_serverless_compute` - Whether this SQL warehouse is a serverless endpoint. If this value is true explicitly or through the default, you **must** also set `warehouse_type` field to `pro`. To avoid ambiguity, especially for organizations with many workspaces, Databricks recommends that you always set this field explicitly.
```

Comment:
as I remember it's not necessary to set it to `PRO` as it's default on the backend

Reply:
See comment thread https://github.com/databricks/terraform-provider-databricks/pull/2249#discussion_r1179487705

Reply:
Per meng, changing this....

```suggestion
* `enable_serverless_compute` - Whether this SQL warehouse is a serverless endpoint. See below for details about the default values. To avoid ambiguity, especially for organizations with many workspaces, Databricks recommends that you always set this field explicitly.
```